src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -410,6 +410,30 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         auto pfnScrollPositionChanged = std::bind(&TermControl::_TerminalScrollPositionChanged, this, std::placeholders::_1, std::placeholders::_2, std::placeholders::_3);
         _terminal->SetScrollPositionChangedCallback(pfnScrollPositionChanged);
 
+        // Set up blinking cursor
+        DispatcherTimer cursorTimer;

```

Comment:
I'd prefer if `cursorTimer` was a private member variable on `TermControl`, and instead of setting up some lambdas here for each of `GotFocus`, and `LostFocus`, we had methods on `TermControl` instead. See the handler for `PreviewKeyDown`, set up on L404 as an example.

Reply:
Thanks for the feedback, I'll work on it!